src/MartinGeorgiev/Doctrine/DBAL/Types/BaseRangeType.php

```
@@ -0,0 +1,71 @@
+/**
+ * Base class for PostgreSQL range types.
+ *
+ * @template T of Range
+ *
+ * @since 3.3
+ *
+ * @author Martin Georgiev <martin.georgiev@gmail.com>
+ */
+abstract class BaseRangeType extends Type
+{
+    public function getSQLDeclaration(array $column, AbstractPlatform $platform): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$column'. (Unused Code Rules)

(UnusedFormalParameter)